client/src/components/EditDetails.jsx

```diff
@@ -1,60 +1,68 @@
-import React from 'react';
-import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
-import '!style-loader!css-loader!bootstrap/dist/css/bootstrap.css';
-import axios from 'axios';
+import React from 'react'
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap'
+import '!style-loader!css-loader!bootstrap/dist/css/bootstrap.css'
+import axios from 'axios'
+
+const Months = Array.from(Array(12)).map(i => i + 1)
+const Days = Array.from(Array(31)).map(i => i + 1)
 
 class EditDetails extends React.Component {
   constructor(props) {
-    super(props);
-    let { height, weight, age, gender, dob, zip } = props.profile;
+    super(props)
+    let { height, weight, age, gender, dob, zip } = props.profile
+
+    const [heightFt, heightIn] = height.split('/')
```

Comment:
Keeping code DRY to a certain degree will improve code readability, maintenance, and less bug. So we can lift `height.split('/')` up and declare the variables just once per function call. This help cuts down the amount of function calls within this code block, which improves performance

https://www.drycode.io/